src/main/java/uk/gov/hmcts/reform/sscs/service/EventService.java

```diff
@@ -101,7 +107,13 @@ private Runnable eventHandler(EventType eventType, SscsCaseData caseData) {
 
     private boolean hasPdfDocument(SscsCaseData caseData) {
         String fileName = emailService.generateUniqueEmailId(caseData.getAppeal().getAppellant()) + ".pdf";
+        LOG.info("Case does has document {} and Pdf file name to check {} ",
```

Comment:
Small typo should be "Case does have document"

Reply:
updated